### ISSUE TITLE
Repeater SiteImprove failures

### ISSUE THREAD

**Issue by jamesjacobs** [jadu/pulsar#1174]
Investigate and fix the follow errors which are caused by `input`'s within `div.repeater__saved-data` not having labels. 

Note these inputs are hidden by a container with `display: none` so it may just be a case of adding `display: none` to the inputs or changing them to `type="hidden"`.

![image](https://user-images.githubusercontent.com/756393/77057897-94dd5700-69cc-11ea-8d23-66a6cf8514ae.png)
